projects/js-packages/storybook/package.json

```diff
@@ -26,10 +26,7 @@
 		"storybook:dev": "concurrently \"pnpm run dev:packages\" \"start-storybook -c ./storybook -p 50240 -s ./public\""
 	},
 	"devDependencies": {
-		"@automattic/jetpack-components": "workspace:^0.10.0",
-		"@automattic/jetpack-connection": "workspace:^0.12.1",
```

Comment:
why remocing these deps?

Reply:
We were getting the stories from `node_modules`, but at the same time, we're building the css from a relative path. I normalized it and get everything from the relative path.